bool req_1
bool req_2
bool req_3
bool full_car_1
bool full_car_0
bool ready_0
bool ready_1